src/toolchain/toolchain.ts

```diff
@@ -368,7 +368,15 @@ export class SwiftToolchain {
                     return undefined;
                 }
                 const data = await fs.readFile(platformManifest, "utf8");
-                const infoPlist = plist.parse(data) as unknown as InfoPlist;
+                let infoPlist;
+                try {
+                    infoPlist = plist.parse(data) as unknown as InfoPlist;
+                } catch (error) {
+                    vscode.window.showWarningMessage(
+                        `unable to parse ${platformManifest}: ${error}`
```

Comment:
Is there any suggestion we can make to help them fix the issue?

Reply:
I don't really know 😢 I think if we were able to detect the encoding that would be good.  However, this is a file that the user doesn't write, it is a distributed file, so its really a bad release.